LanguageFeatures/Patterns/constant_A03_t06.dart

```diff
@@ -34,42 +34,41 @@ import "../../Utils/expect.dart";
 
 String testList(List value) {
   return switch (value) {
-    case const <int>[1, -2] => "<int>[1, -2]";
-    case const <dynamic>[1, -2] => "<dynamic>[1, -2]";
-    case const <String>["3", "4"] => "<String>['3', '4']";
-    case const <dynamic>["3", "4"] => "<dynamic>['3', '4']";
-    case const <double>[] => "<double>[]";
-    case const <dynamic>[] => "<dynamic>[]";
-    default => "default";
+    const <int>[1, -2] => "<int>[1, -2]",
+    const <dynamic>[1, -2] => "<dynamic>[1, -2]",
+    const <String>["3", "4"] => "<String>['3', '4']",
+    const <dynamic>["3", "4"] => "<dynamic>['3', '4']",
+    const <double>[] => "<double>[]",
+    const <dynamic>[] => "<dynamic>[]",
+    _ => "default"
   };
 }
 
 String testMap(Map value) {
   return switch (value) {
-    case const <int, int>{1: -2} => "<int, int>{1: -2}";
-    case const <dynamic, dynamic>{1: -2} => "<dynamic, dynamic>{1: -2}";
-    case const <String, num>{'answer': 42} => "<String, num>{'answer': 42}";
-    case const <dynamic, dynamic>{'answer': 42} =>
-        "<dynamic, dynamic>{'answer': 42}";
-    case const <String, Object>{'true': true} =>
-        "<String, Object>{'true': true}";
-    case const <dynamic, dynamic>{'true': true} =>
-        "<dynamic, dynamic>{'true': true}";
-    case const <String, int>{} => "<String, int>{}";
-    case const <dynamic, dynamic>{} => "<dynamic, dynamic>{}";
-    default => "default";
+    const <int, int>{1: -2} => "<int, int>{1: -2}",
+    const <dynamic, dynamic>{1: -2} => "<dynamic, dynamic>{1: -2}",
+    const <String, num>{'answer': 42} => "<String, num>{'answer': 42}",
+    const <dynamic, dynamic>{'answer': 42} =>
+      "<dynamic, dynamic>{'answer': 42}",
```

Comment:
Does the formatter handle this code at all? If it doesn't then it is probably safest to use an extra indent of 4 relative to the `=>` line (similarly to `=>` functions, and initializing expressions in declarations, and continued expressions). If it does work, and it uses an extra indent of 2 then that's the way to go, of course.

Reply:
This should be safe, for any big switch expressions the formattor should treat the case with a trailing comma and the case without a trailing comma the same (they differ by that trailing comma, but they don't differ by the indentation level and other, larger things). So let's stop worrying. ;-) Landing.